migration-guides/v5.0.0.md

```diff
@@ -0,0 +1,17 @@
+# v5.0.0 Migration Guide
+
+`FlashMessage` now expects separate `children` and `isError` props, rather than a single `message` prop. This allows the component to be used without needing to use `redux-flash`.
+
+You should update any instances of `FlashMessage` accordingly.
+
+(Note: this change is not necessary if you are using `FlashMessageContainer` instead of `FlashMessage` directly.)
+
+```jsx
+
+// Before, where "message" was a redux-flash message object:
+<FlashMessage message={message} />
+
+// After:
+<FlashMessage isError={message.isError} {message.props}>{message.message}</FlashMessage>
```

Comment:
Can you explain what `{message.props}` is accomplishing here?

Reply:
That's a typo- let me know if it's still unclear.

Reply:
I could not wrap my brain around that haha. Makes sense now!